language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/TypeTestExpressionNodeContext.java

```diff
@@ -98,6 +98,7 @@ public TypeTestExpressionNodeContext() {
             return completionItems;
         }
         completionItems = typeReferences.stream().filter(typeRef -> isQualifiedTypeReference(context, typeRef))
+                .map(CommonUtil::getRawType)
```

Comment:
This change was added in https://github.com/ballerina-platform/ballerina-lang/pull/34317. Can you rebase this PR with the tracking branch and apply your change on top of that?